osu.Game/Screens/SelectV2/SongSelect.cs

```diff
@@ -179,6 +161,41 @@ private void load()
                                     },
                                 }
                             },
+                            new GridContainer // used for max width implementation
```

Comment:
Why is this split out from the other `GridContainer`? Again, something being done with no documentation on why.